### DISCUSSION TITLE
Price Converter not found in the repository

### DISCUSSION THREAD

**Discussion by Tarreef** [Cyfrin/foundry-full-course-cu#2747] [Q&A]
Please, i can't see the same price converter file Mr. Patrick is working on in the course...
The one i saw in DataFeeds on chain.link is different.
So, am i still allowed to use that one instead

**Comment by EngrPips** [comment]
Hey @Tarreef, Did you figure it out already?

**Comment by Alsouuuu** [comment]
https://docs.chain.link/data-feeds/using-data-feeds
